### PR TITLE
Added data loader for schools and sponsors (Organizations)

### PR DESCRIPTION
**Pull request by clintonb**
ECOM-5191 and ECOM-5192

### CODE REVIEW

**Comment by clintonb** [comment]
@edx/ecommerce please review.

File: course_discovery/apps/course_metadata/data_loaders/tests/test_marketing_site.py [hunk 1]
```diff
@@ -415,3 +415,47 @@ def test_ingest(self):
...
+        self.assertEqual(sorted(school.tags.names()), ['charter', 'founder'])
+
+    @responses.activate
+    def test_ingest(self):
+        self.mock_login_response()
+        api_data = self.mock_api()
+
+        self.loader.ingest()
+
+        for datum in api_data:
```
Comment (rlucioni) [inline course_discovery/apps/course_metadata/data_loaders/tests/test_marketing_site.py]: Minor, but would read better as `for school in schools`.

File: course_discovery/apps/course_metadata/models.py [hunk 2]
```diff
@@ -156,11 +157,14 @@ class Organization(TimeStampedModel):
     uuid = models.UUIDField(blank=False, null=False, default=uuid4, editable=False, verbose_name=_('UUID'))
     key = models.CharField(max_length=255)
     name = models.CharField(max_length=255)
+    marketing_url_path = models.CharField(max_length=255, null=True, blank=True)
```
Comment (rlucioni) [inline course_discovery/apps/course_metadata/models.py:160]: Could this be a URLField?

Comment (rlucioni) [inline course_discovery/apps/course_metadata/models.py:160]: Maybe not, since it's just a path.

**Comment by rlucioni** [comment]
👍

File: course_discovery/apps/course_metadata/data_loaders/tests/test_marketing_site.py (hunk 1 above)
Comment (clintonb) [inline course_discovery/apps/course_metadata/data_loaders/tests/test_marketing_site.py]: Updated.

**Comment by clintonb** [comment]
FYI @bderusha I have added a sponsor data loader.

**Pull request merged**